verde/base/base_classes.py

```diff
@@ -234,7 +240,7 @@ def grid(
         data_names=None,
         projection=None,
         **kwargs
-    ):
+    ):  # pylint: disable=too-many-locals
```

Comment:
@leouieda Do you agree with adding this exception?

Reply:
Yes but I would like to try not making a habit of this. I've been frustrated with this as well but I like how it forces me to really think about the function.

But in this case, this is fine. We're planning refactoring of `grid` soon anyways and can remove that when we do it.